src/components/Modal/CartModal/CartModal.tsx

```diff
@@ -1,39 +1,97 @@
 import FormBlock from './FormBlock/FormBlock'
+import { CartState } from '../../../types/state'
+
+const sendMailOrder = async (
+  data: Record<string, string | boolean>,
+  products: CartState['groupedByToppingsProducts'],
+  price: number,
+  submitLocal: string,
+  isValidLocal: boolean
+) => {
+  try {
+    await fetch('/api/order', {
+      method: 'POST',
+      headers: { 'content-type': 'application/json' },
+      body: JSON.stringify({
+        data: {...data, price},
+        products,
+        submitLocal,
+        isValidLocal,
+      }),
+    })
+  } catch(error){
+    return 0
+  }
+}
 
 const CartModal = () => {
-  const { register, handleSubmit, formState: { errors } } = useForm()
+
+  const { register, handleSubmit, setValue, formState: { errors } } = useForm()
 
   const { state: { show }, actions: { closeModal } } = useContext(ModalContext)
   const { state: { price, groupedByToppingsProducts } } = useContext(CartContext)
 
-  const [orderType, setOrderType] = useState<OrderType>('')
+  const local = window.location.host.split('.')[0]
+  const isValidLocal = local === VALID_LOCALS.botanica || local === VALID_LOCALS.rascanovca
+  const inputValidations = isValidLocal ? VALIDATIONS.masa : VALIDATIONS.email
+
+  const [orderType, setOrderType] = useState<OrderType>('takeaway')
+  const [orderPayment, setOrderPayment] = useState<OrderPayment>('')
+
+  const { t } = useTranslation('cart')
 
   const onSubmit = (data: Record<string, string | boolean>) => {
-    // eslint-disable-next-line no-console
-    console.table(data)
+
+    const getTakeawayLocation = (location: string) => {
+      switch (location){
+        case 'Telecentru' :
+          return 'manager.mister.'
+        case 'Malina Mică' :
+          return 'manager.mister.'
+        case 'Râșcani' :
+          return 'rascanovca.mr.'
+        case 'Botanica' :
+          return 'botanica.mr.'
+        default :
+          return 'manager.mister.'
+      }
+    }
+
+    sendProductsToCMS(groupedByToppingsProducts, data, price)
+
+    const submitLocal = isValidLocal ? `${local}.mr.` : getTakeawayLocation(data.takeawayLocation as string)
+
+    sendMailOrder(data, groupedByToppingsProducts, price, submitLocal, isValidLocal)
   }
 
   const isThroughDelivery = orderType === ORDER_TYPE.delivery
 
-  const CurrentShowedBlock = orderType ? CART_FORM_COMPONENTS[orderType] : null
-
-  const orderTypeRadio = ({ text, type }: typeof ORDER_TYPE_BUTTONS[0], index: number) =>
-    <RadioButton key={index} text={text} selected={orderType === type} onChange={() => setOrderType(type)} />
+  const CurrentShowedBlock = isValidLocal ? null : orderType ? CART_FORM_COMPONENTS[orderType] : null
+  const CurrentShowedInput = isValidLocal ? ORDER_FROM_INPUTS[ORDER_FROM.local] : ORDER_FROM_INPUTS[ORDER_FROM.in_afara_localului]
+  const CurrentShowedForm = isValidLocal ? ORDER_FROM_FORMS[ORDER_FROM.local] : ORDER_FROM_FORMS[ORDER_FROM.in_afara_localului]
```

Comment:
add ternary in barces, like this:
```
const { local, in_afara_localului } = ORDER_FROM
...
const CurrentShowedForm = ORDER_FROM_FORMS[isValidLocal ? local : in_afara_localului]
```
and same for `CurrentShowedInput `

Reply:
Da pentru ce sa fac const { local, in_afara_localului } = ORDER_FROM?
Nu pot prosta sa adaug ca string in 
ORDER_FROM_FORMS[isValidLocal ? 'local' : 'in_afara_localului']